docs/application/dotnet/samples/appfw/overview.md

```diff
@@ -13,17 +15,31 @@
 			<td><img alt="" height="180" src="media/walarm_list.png" width="180"/></td>
 			<td>
 			<p><strong>(W) Alarm</strong> [In progress]</p>
-			<p>This sample application demonstrates how you can create and manage several alarms, and save alarm data.
-            Also, this sample demonstrates how you can create an application using Xamarin.Forms, Tizen.Wearable.CircularUI.Forms.</p>
-			<p><a href="https://developer.tizen.org/development/sample/native/UI/%28Circle%29_Alarm" target="_blank">Native version</a></p>
-			<p><a href="https://developer.tizen.org/development/sample/web/UI/Alarm_UI" target="_blank">Web version</a></p>
+			<p>This sample application demonstrates how to create and manage several alarms and save the alarm data. It also demonstrates how to create an application using Xamarin.Forms and Tizen.Wearable.CircularUI.Forms.</p>
+			<p>In addition, there are similar native and web sample applications.<br>
+			<a href="https://developer.tizen.org/development/sample/native/UI/%28Circle%29_Alarm" target="_blank">Native version</a><br>
+			<a href="https://developer.tizen.org/development/sample/web/UI/Alarm_UI" target="_blank">Web version</a></p>
 			</td>
-		</tr>		
+		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m1appcommon.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/AppCommon" target="_blank"><strong>(M) App-common</strong></a></p>
-			<p>This sample application demonstrates how to work with app-common API in order to obtain application-specific information.</p>
+			<p>This sample application demonstrates how to work with app-common APIs to obtain application-specific information.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/wappinformation.png" width="180"/></td>
+			<td>
+			<p><strong>(W) AppInformation</strong> [In progress]</p>
+			<p>This sample application demonstrates how to get application-specific information(e.g. AppId, Package ID, shared directory path) and directory information(resource, cache, app data and so on).</p>
```

Comment:
Suggestion:
Can we rephrase as:
This sample application demonstrates how to get the application-specific information (AppId, Package ID, and shared directory path) and the directory information (resource, cache, app data, and so on).
OR
This sample application demonstrates how to get the application-specific information such as AppId, Package ID, and shared directory path. It also demonstrates how to get the directory information such as resource, cache, app data and so on.

Please verify and update.

Reply:
done